src/api/app/services/diff_parser.rb

```diff
@@ -47,9 +48,13 @@ def generate_inline_diffs
       block.zip(processed_lines) do |line, content|
         sign = line.state == 'added' ? '+' : '-'
         line.content = "#{sign}#{content}" unless full_line_diff(content)
+        # Known scenario https://github.com/openSUSE/open-build-service/pull/17006
+        # Log if a different unexpected scenario happen instead
+        Rails.logger.warn "generate_inline_diffs: content is nil - line='#{line.to_json}'" if content.nil? && line.content != '+'
```

Comment:
I would move it to before we modify the lines

Reply:
Done

Reply:
What are we logging this for?